src/main/java/com/server/sopt/seminar/dto/Response.java

```diff
@@ -0,0 +1,16 @@
+package com.server.sopt.seminar.dto;
+
+import lombok.Getter;
+
+@Getter
+public class Response {
+    private int code;
+    private String status;
+    private boolean success;
+
+    public Response(int code, String status, boolean success){
+        this.code = code;
+        this.status = status;
+        this.success = success;
+    }
+}
```

Comment:
생성자 대신 @Builder 애노테이션과 정적 팩토리 메서드 패턴을 적용해서 코드를 리팩터링 해봐도 좋을 것 같습니다.

정적 팩토리 메서드 패턴을 적용해서 객체 생성을 제한하면 다음과 같은 이점이 있습니다.
1. 메서드명으로 생성 목적에 대해 명확하게 명시할 수 있어서 반환될 객체의 특성을 정확히 표현할 수 있습니다.
ex) Member 클래스가 존재할 때, createMember()라는 이름을 지어 개발자의 정확한 의도를 나타낼 수 있습니다. 
2. 정적 메서드만으로 객체 생성을 제한하기 때문에 인스턴스에 대한 통제 및 관리에 용이합니다. 따라서 불필요하게 객체가 생성되는 것을 방지할 수 있습니다.
3. 다형성을 활용하여 하위 자료형 객체를 반환할 수 있어 유연성이 높아집니다.
4. 적절한 전처리 작업을 추가하여 개발자의 의도에 맞게 반환값을 조절할 수 있습니다.
5. 객체 생성을 캡슐화하여 내부 구현을 숨길 수 있습니다. 또한 인터페이스 또는 추상 클래스를 반환값으로 활용하여 특정 구현체에 대한 의존성을 제거할 수 있습니다.

```suggestion
@Builder(access = AccessLevel.PRIVATE)
@AllArgsConstructor(access = AccessLevel.PRIVATE)
@Getter
public class Response {
    private int code;
    private String status;
    private boolean success;
    
    public static Response of(int code, String status, boolean success) {
       return Response.builder()
               .code(code)
               .status(status)
               .success(success)
               .build();
    }
}
```

Reply:
우와 자세한 설명 감사합니다 참고해서 수정해볼게요!